src/mobot_client/models/__init__.py

```diff
@@ -0,0 +1,505 @@
+#  Copyright (c) 2021 MobileCoin. All rights reserved.
```

Comment:
Are the models supposed to live in the __init__.py? My understanding looking at Django best practices is that they should live in a models.py (https://django-best-practices.readthedocs.io/en/latest/applications.html)

Reply:
Normally, yes, but putting them in a models module is fine if there's split up files in it, which we have. Django is fine with either.